sdks/python/apache_beam/runners/common.py

```diff
@@ -402,7 +402,7 @@ def __init__(self,
     self.window_fn = windowing.windowfn
 
     self.do_fn_invoker = DoFnInvoker.create_invoker(
-        do_fn_signature, context, side_inputs, args, kwargs)
+        self, do_fn_signature, context, side_inputs, args, kwargs)
```

Comment:
Got it. I think it'll be good to rename it to `process_output` if we don't want to go down the function pointer router.